bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/HeapSizeManager.java

```diff
@@ -183,5 +189,10 @@ public void onFailure(Throwable t) {
 
   public void markCanBeCompleted(Long id) {
     completedOperationIds.offer(id);
+    if (this.isFlushing.get() && completedOperationIds.size() == pendingOperationsWithSize.size()) {
```

Comment:
I understand "this.isFlushing.get()" means some thread is in flush().
What does it mean when completedOperationIds.size() == pendingOperationsWithSize.size()?